po/cs.po

```diff
@@ -2015,7 +2015,7 @@ msgstr "<b>%d</b>"
 #: ../virtaal/views/propertiesview.py:203
 #, python-format
 msgid "%.1f KB"
-msgstr "%.1f kB"
+msgstr "%.1f KB"
```

Comment:
The ACRONYM filter is dumb.  It is simply flagging potential errors.  kB is perfectly correct in some locales, in fact our source could be considered incorrect.